src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -103,9 +104,10 @@ public void storeMessage(Message message) {
         String user = (String) entity.getProperty("user");
         String text = (String) entity.getProperty("text");
         long timestamp = (long) entity.getProperty("timestamp");
-        String recipient = (String) entity.getProperty("recipient");
+        String recipient = (String) entity.getProperty("recipient"); 
+        float sentimentScore = entity.getProperty("sentimentScore") == null? (float) 0.0 : ((Double) entity.getProperty("sentimentScore")).floatValue();
```

Comment:
Could you explain why you're casting the sentiment score to Double from float, and then save it in a float variable